test/monarch/camp_monarch_interface.F90

```diff
@@ -3,15 +3,14 @@
 ! SPDX-License-Identifier: MIT
 
 !> \file
-!> The monarch_interface_t object and related functions
+!> The camp_monarch_interface_t object and related functions
 
-!> Interface for the MONACH model and PartMC-camp
-module camp_monarch_interface
+!> Interface for the MONACH model and CAMP-camp
+module camp_monarch_interface_2
```

Comment:
why was this renamed?

Reply:
Yeah, that is not needed, I passed so much time with 2 `camp_monarch_interface` that I get used to that and don't even think it can be renamed.